src/settings_gui.cpp

```diff
@@ -229,6 +231,7 @@ struct GameOptionsWindow : Window {
 						 * entries in the dropdown list. */
 						item->SetParamStr(0, _languages[i].name);
 					}
+					item->SetParam(1, (LANGUAGE_TOTAL_STRINGS - _languages[i].missing) * 100 / LANGUAGE_TOTAL_STRINGS);
```

Comment:
Are there no issues with setting this parameter then not using it (in the case of `STR_JUST_RAW_STRING` ) ?

Reply:
RAW_STRING parameters are only borrowed pointers. The caller has to make sure, they remain valid, as long as anyone processed the string.
In this case the optional parameter is a NUM, not a RAW_STRING.